index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio Webpage</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li>
+          <a href="#welcome-section">Projects</a>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio</h1>
+    </section>
+
+    <a
+      rel="noopener"
+      id="profile-link"
+      href="https://github.com/omarbabou"
+      target="_blank"
+    ></a>
```

Comment:
You can comment about this `<a>` tag currently it's not included inside any proper `tag`. Kindly, mention this to follow [semantic HTML tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.) rules.